packages/reports/tests/integration/components/navi-request-preview-test.js

```diff
@@ -183,5 +206,52 @@ module('Integration | Component | navi-request-preview', function(hooks) {
 
     await click('.navi-request-preview__column-header--metric>.navi-request-preview__column-header-sort');
     await click('.navi-request-preview__column-header--metric:last-of-type>.navi-request-preview__column-header-sort');
+
+    //Edit the revenue metric column
+    await click(
+      findAll('.navi-request-preview__column-header--metric')
+        .find(el => el.textContent.trim() === 'Revenue (USD)')
+        .querySelector('.ember-basic-dropdown-trigger')
+    );
+    await click('.navi-request-preview__column-header-option:last-of-type');
+
+    assert
+      .dom('.navi-request-column-config__parameter')
+      .isVisible('Parameter option is available for parameterized metric');
+    assert
+      .dom('.navi-request-column-config__parameter-trigger')
+      .hasText('Dollars (USD)', 'Current parameter is selected');
+    assert.dom('#columnName').hasValue('Revenue (USD)', 'Column name is shown in input box');
+
+    await selectChoose('.navi-request-column-config__parameter-trigger', 'Dollars (CAD)');
+    assert.dom('#columnName').hasValue('Revenue (CAD)', 'Column name updates with metric param');
+    assert.deepEqual(
+      textContentArray('.navi-request-preview__column-header'),
+      ['Date', 'Age', 'Banner Ad Clicks', 'Ad Clicks (VideoAds)', 'Revenue (CAD)'],
+      'Column name updates in column headers'
+    );
+
+    await fillIn('#columnName', "Dolla Dolla Bill Y'all");
```

Comment:
Maybe use [`typeIn`](https://github.com/emberjs/ember-test-helpers/blob/master/API.md#typein) in the future?